style.css

```diff
@@ -189,3 +204,74 @@ footer {
   margin-bottom: 24px;
   padding: 12px 0 0 12px;
 }
+
+@media only screen and (min-width: 768px) {
+  .links {
+    display: flex;
+    list-style-type: none;
+    width: 270px;
+    justify-content: space-between;
+  }
+
+  .links li > a {
+    text-decoration: none;
+    color: white;
+    font-family: Inter, sans-serif;
+    font-size: 15px;
+    font-weight: 600;
+    line-height: 20px;
+    letter-spacing: 0;
+    text-align: center;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .headline > div {
+    width: 100%;
+    max-width: 1155px;
+    flex-direction: row-reverse;
+  }
+
+  .headline-title {
+    max-width: 566px;
+  }
+
+  .headline-description {
+    max-width: 566px;
+  }
+
+  .headline-social {
+    flex-direction: column;
+    justify-content: space-between;
+    height: 147px;
+    width: auto;
+    margin-right: 3%;
+    align-self: center;
+  }
+
+  .headline {
+    flex-direction: row-reverse;
+    background: url("./images/Illustration-header2.png") #1c1a19 no-repeat;
+    background-position: 100% 50%;
+    background-size: 30%;
+  }
+
+  .contact-form {
+    flex-direction: row;
+  }
+
+  .form-text {
+    text-align: start;
+    max-width: 455px;
+  }
```

Comment:
- [ ] Please stick to the Figma design your form heading is not the same as the Figma design so kindly fix it :+1: check below :point_down:  the comparison of your design and Figma design :slightly_smiling_face: 

**Your design**
![Screenshot from 2023-03-30 00-26-37](https://user-images.githubusercontent.com/45226848/228647801-622592c1-15ac-47dd-8e98-d672d055d09e.png)

**Figma design**
![Screenshot from 2023-03-30 00-26-21](https://user-images.githubusercontent.com/45226848/228647780-071a2dca-07ec-4ace-8fd7-f37372a26da0.png)

- [ ] So if you see your design the `text-align` of your `heading`  and Figma design `heading` is not the same. Also, please remove `three dots` after `something text` its not look professional so kindly fix all these changes. :+1: